Documentation/contributing/development/reviewers_committers/triage.rst

```diff
@@ -0,0 +1,138 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _triage:
+
+Triaging Issues
+===============
+
+The Triager role involves a one week rotation of looking at incoming
+bugs in Cilium and triaging them, as well as helping with merging and
+testing external contributions.
+
+Tasks
+-----
+Tasks are listed by order of importance.
+
+1. Merging and Pushing Contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As the title indicates, this part involves looking at community contributions
+and seeing them through to completion. This involves approving workflows and
+calling ``/test``, pulling in code owners and, ultimately, merging the PR when
+it is ready. The amount of effort put on these should be proportional to the
+effort put in by the contributor.
+
+If a (somewhat) large contribution comes in that requires long-term help, the
+Triager takes care of pairing up the contributor with someone from
+the corresponding area, who can then help guide the
+contribution even if it takes several weeks.
+
+PRs that are ready to merge
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`cilium/cilium: ready to merge PRs
+<https://github.com/cilium/cilium/pulls?q=is%3Aopen+is%3Apr+label%3Aready-to-merge>`_
+
+Guidelines for merging:
+
+*   You should basically never be bypassing branch protections, even for
+    docs-only PRs. It's better to run ``/test`` regardless, and that should be
+    able to determine which tests are required. People should not be setting
+    ``ready-to-merge`` manually, we should always run ``/test``. If that runs
+    more CI than it should, that is a problem with CI.
+*   Once all required tests have passed, reviews are in, and discussions are
+    resolved, have a maintainer push the merge button without needing to bypass branch
+    protections. The button doesn't have to be green.
+*   `Maintainer's Little Helper
+    <https://github.com/cilium/maintainer-little-helper>`_ does not, at the
+    time of writing, know about the requirement to resolve discussions, so it
+    may flag PRs as ready-to-merge even when you can't click the merge button.
+    In this case, review the discussions and see if there are any outstanding
+    discussions, and if not, close them.
```

Comment:
This stuff should probably belong in https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#getting-a-pull-request-merged and just be referenced there. Maybe drop the text for now / defer to another PR?